pyproject.toml

```diff
@@ -135,4 +135,4 @@ score = "no"
 
 [tool.hdev]
 project_name = "tox-faster"
-project_type = "package"
+project_type = "library"
```

Comment:
Another bug fix: `"package"` should always have been `"library"`. (We'll probably end up removing this from `pyproject.toml` at some point anyway since it's only for `hdev`.)